eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -202,6 +203,11 @@ instance
 
   fixupTx = alonzoFixupTx
 
+  scriptTestContexts =
+    plutusTestScripts SPlutusV1
+      <> plutusTestScripts SPlutusV2
```

Comment:
I think there might be a problem with his approach, since  PlutusV1 and PlutusV2 scripts are identical, their hash will be identical, which will result in TestContexts for PlutusV2 to disappear. 

That'll probably work out for now, but might become a problem in the future. So, I don't suggest changing the implementation right now, but adding a comment on the `scriptTestContexts` would be useful for the future that this how it works right now.

Reply:
It's probably better to make an issue for this, otherwise nobody is going to notice this. I think it'd make sense to have the key be `(ScriptHash (EraCrypto era), Language)`